modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -73,6 +80,53 @@ const AddFDSFieldsModalContent = ({
 		);
 	};
 
+	const synchronizeFDSFields = async () => {
```

Comment:
I'm not personally too sticky to this thing, but naming here may be misleading as we're creating an AddXModal which ends up calling SyncX function. Nothing that can't be changed later on.

Reply:
Yeah, I agree it's not ideal. The whole UI may be misleading, the component name is just a symptom. A `(+)` button implies creation of an item, not the sync (create + delete) functionality we are implementing. @ugeortiz Is there maybe a better UI pattern, or is this good enough?